src/applications/form-renderer/tests/unit/utils/formConfig.unit.spec.js

```diff
@@ -103,4 +108,41 @@ describe('createFormConfig', () => {
       });
     });
   });
+
+  context('with Identification Information pattern', () => {
+    let serviceNumberIncluded;
+    let vetIdOnly;
+
+    beforeEach(() => {
+      serviceNumberIncluded = createFormConfig(employmentQuestionnaire)
+        .chapters[20002].pages[20002];
```

Comment:
This `.chapters[20002].pages[20002]` reminds me that I still think we need to discuss this chapters/pages thing further (as a whole team).